app/config.py

```diff
@@ -512,6 +512,7 @@ class Config(object):
 
     # Feature flags for bounce rate
     FF_BOUNCE_RATE_V1 = env.bool("FF_BOUNCE_RATE_V1", False)
+    FF_BOUNCE_RATE_SEED_EPOCH_TIMESTAMP = os.getenv("FF_BOUNCE_RATE_SEED_TIMESTAMP", False)
```

Comment:
I'm not seeing `FF_BOUNCE_RATE_SEED_EPOCH_TIMESTAMP` anywhere else in this PR - is that intentional?

Also 2 suggestions:
- let's make the variable names match (`FF_BOUNCE_RATE_SEED_EPOCH_TIMESTAMP` vs `FF_BOUNCE_RATE_SEED_TIMESTAMP`), since that is the convention for the other env variables and it makes it easier to find them in the project
- can we use a name for the FF that will make it more obvious what it's doing? maybe `FF_CALCULATE_BOUNCE_RATE` or just `FF_BOUNCE_RATE` if this is for the whole feature

Reply:
ah this was for the latest commit that I hadn't pushed up. I just pushed. Happy to change the name though its not a flag... its actually epoch. So unsure if we want to say FF for a non-boolean.